recipe/meta.yaml

```diff
@@ -17,18 +17,21 @@ requirements:
     - setuptools
     - cython >=0.23
     - python
-    - six >=1.4
+    - six >=1.7.3
     - numpy x.x
-    - scipy >=0.9
+    - scipy >=0.17
+    - numpydoc >=0.6
   run:
     - python
-    - six >=1.4
-    - numpy x.x
```

Comment:
Yep, looks like you handled this correctly. Sorry was AFK so couldn't respond further. Please let me know if you have more questions about this.